src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -3246,6 +3247,21 @@ <h3>
         </a> for more.
     </p>
     <!--  ****************** -->
+    <h3>
+        <a id="FlameGraphView">Flame Graph View</a>
+    </h3>
+    <p>
+        The flame graph view shows the same data as call tree view, but using different visualization.&nbsp;&nbsp;&nbsp;
+        It gives you very intelligible overview.&nbsp;&nbsp;
+        The graph starts at the bottom. Each box represents a method in the stack. Every parent is the caller, children are the callees.
+        The wider the box, the more time it was on-CPU. The samples count is shown in the tooltip and in the bottom panel.
+        To change the content of the flame graph you need to apply the filters for call tree view.&nbsp;&nbsp;
```

Comment:
Can you put a hyperlink to some documentation on how to use FlameGraphs that already exists somewhere on the internet?